python/tank/bootstrap/cached_configuration.py

```diff
@@ -194,6 +194,18 @@ def update_configuration(self):
 
         # copy the configuration into place
         try:
+
+            # first do a quick validation check that the descriptor payload looks like a config
+            # ensure that there is a core folder in the config
+            self._descriptor.ensure_local()
+            path_to_descriptor_payload = self._descriptor.get_path()
+            if not os.path.exists(os.path.join(path_to_descriptor_payload, "core")):
```

Comment:
Why? `tk-config-basic` only has `roots.yml` and `core_api.yml`, and looking at the code both of these files are optional. It would seem someone who is not interested into specifying any of these two should be allowed to not provide a core folder. Especially if the config is stored in git, in which case we'll be forcing people to create a folder and add a placeholder file to please git.